cmd/shield/raw.go

```diff
@@ -14,3 +14,8 @@ func RawJSON(raw interface{}) error {
 	fmt.Printf("%s\n", string(b))
 	return nil
 }
+
+func RawUUID(uuid string) error {
+	fmt.Printf("%s", uuid)
```

Comment:
Can you tack a `\n` onto the end of this printf?  I believe that `$(...)` will strip the trailing whitespace for you, and it makes for a less jarring experience when experimenting with flags in an interactive shell.